### DISCUSSION TITLE
研究生模板内，章节标题\chapter 的字号大小问题

### DISCUSSION THREAD

**Discussion by urnotLeo** [nju-lug/NJUThesis#106] [Q&A] [answered]
看师兄师姐的模板里，章节标题的大小通常会大一点，
![微信截图_20220323191231](https://user-images.githubusercontent.com/35950564/159686812-47ba3c06-1d89-4ade-b901-90337236f891.png)

而当前模板中的章节标题字号较小，请问可以通过什么方法调整一下吗？

**Comment by atxy-blip** [comment, marked as answer]
这个是 `ctex` 宏集提供的基础设置，手册见 [ctex.pdf](http://mirrors.ctan.org/language/chinese/ctex/ctex.pdf)。

可以在导言区（或者任何你喜欢的位置）仿照下面的命令进行调整。

```TeX
\ctexset{
  chapter/format = \bfseries\rmfamily\zihao{3},
  section/format = \bfseries\rmfamily\zihao{4}
}

**Reply by urnotLeo** [reply]
好的！谢谢~
